src/features/currentTodo.ts

```diff
@@ -1,10 +1,17 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { PayloadAction, Slice, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 const initialState = null as Todo | null;
```

Comment:
typing initialState can make the code more readable

```suggestion
type CurrentTodoState = Todo | null;

const initialState: CurrentTodoState = null;
```

Reply:
I corrected all the comments except this one, I don’t know why, but when I write like this, typing problems arise, I tried many different typing options, but the problem always remained
![image](https://github.com/user-attachments/assets/213a7f62-a251-4471-9087-c1938b9b0ae4)